### DISCUSSION TITLE
I love your drawings! Where can i get em?

### DISCUSSION THREAD

**Discussion by ProggePal** [roipeker/graphx#65] [General]
For real, I love the way they bounce and disappear. Can you release the source code? 😇

https://giphy.com/gifs/f6UJj36HqFYJuejz5M

https://giphy.com/gifs/uliHRVWVW5IlliliIi

**Comment by roipeker** [comment]
Thanks for the kind comment @Paalpe ! 

Sadly most demos sources are long lost, like temporal experiments i used to overwrite as i was playing while building graphx, or buried inside some really cryptic folder structure backup in some external hd, mangled with other unrelated flutter stuffs ... so most gifs were part of the package building process, sorry for not being able to help you on this mate.
